Makefile

```diff
@@ -28,6 +28,14 @@ FEDORA_IMAGE_REPO?=quay.io/security-profiles-operator/$(IMAGE_NAME)-fedora:$(IMA
 
 TEST_OS?=fedora
 
+SEMODULE_BACKEND?=semanage
```

Comment:
thought you wanted to default to `policycoreutils` already

Reply:
Initially I wanted to only change centos builds, because I think that fedora builds are fixed, but a) I have not verified that outside a local VM and b) it makes sense to keep one default. So, done.